### PR TITLE
feat(app): add liquid classes values function

### PR DESCRIPTION
**Pull request by koji**
# Overview
adding functions to get liquid classes values (replicate PD's implementation) and activate Reset buttons for Aspirate/Dispense.

fyi
from the default value, a few screen's keyboards aren't correct(ex delay's keyboard doesn't support decimal point even thougth the default value is a decimal number) and there issue will be resolved by a following pr.

close AUTH-1011 and AUTH-2000

## Test Plan and Hands on Testing
1. create a new quick transfer
check values are applied to aspirate and dispense
2. modify the values in aspirate/dispense
3. reset values in aspirate/dispense
check the values are reset

## Changelog
- add a new util to retrieve liquid class values and add a boolean to QuickTransferSummaryState to handle apply the liquid class values to aspirate and dispense.

## Risk assessment
mid

### CODE REVIEW

File: app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+
+const convertBlowoutLocation = (
+  location: string | undefined,
+  state: QuickTransferSummaryState
+): BlowOutLocation | undefined => {
+  if (!location) return undefined
+
+  switch (location) {
+    case 'source':
+      return 'source_well'
```
Comment (ncdiehl11) [inline app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts]: I think we have constants for these in `step-generation`

File: app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+    case 'destination':
+      return 'dest_well'
+    case 'trash':
+      return state.dropTipLocation
+    default:
+      return undefined
+  }
+}
```
Comment (ncdiehl11) [inline app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts]: not sure if this is a bit cleaner:

```suggestion
const convertBlowoutLocation = (
  state: QuickTransferSummaryState,
  location?: string
): BlowOutLocation | undefined => {
  switch (location) {
    case 'source':
      return 'source_well'
    case 'destination':
      return 'dest_well'
    case 'trash':
      return state.dropTipLocation
  }
}
```

File: app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+    volume,
+    destinationWells,
+  } = state
+
+  const allLiquidClassDefs = getAllLiquidClassDefs()
+  const liquidClassMap = new Map<string, string>([
+    ['water', WATER_LIQUID_CLASS_NAME],
+    ['glycerol_50', GLYCEROL_LIQUID_CLASS_NAME],
+    ['ethanol_80', ETHANOL_LIQUID_CLASS_NAME],
+  ])
```
Comment (ncdiehl11) [inline app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts:258]: is map needed here since our keys are just the liquid class names?

File: protocol-designer/src/steplist/formLevel/handleFormChange/utils.ts [hunk 4]
```diff
@@ -432,6 +432,8 @@ const getSubmergeRetractFields = (args: {
     isConditioningVolumeEnabled = false,
   } = args
 
+  console.log('additionalEquipmentEntities', additionalEquipmentEntities)
```
Comment (ncdiehl11) [inline protocol-designer/src/steplist/formLevel/handleFormChange/utils.ts]: can remove

**Review by ncdiehl11 — COMMENTED** [review]
looks good, still making my way through. couple of small comments

File: app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts (hunk 3 above)
Comment (koji) [reply app/src/organisms/ODD/QuickTransferFlow/utils/retrieveLiquidClassValues.ts:258]: yes, liquid class is just name and need to align it with the proper name for the following lines.

**Comment by koji** [comment]
- [x] address feedback
- [x] update the values for the latest ui/state update

**Review by ncdiehl11 — APPROVED** [review]
This works well and seems to follow the logic I have on the PD side. I will approve and continue to test edge cases
